subsystem/swerveIntakePivotController.py

```diff
@@ -0,0 +1,93 @@
+import commands2
```

Comment:
Should this be in this changeset or is swerveIntakePivotController from another branch? If the latter, we can help get this cleaned up so this is ONLY for telemetry.